examples/aiohttp/README.md

```diff
@@ -0,0 +1,21 @@
+# Aiohttp autoinstrumented application
+
+This is a barebone aiohttp app used for demonstrating autoinstrumentation with EDOT.
+
+You can build the application image it with:
+
+```
+docker build --load -t edot-aiohttp:latest .
+```
+
+You can run the application with:
+
+```sh
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.es.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer P....l"
+docker run -e OTEL_EXPORTER_OTLP_ENDPOINT="$OTEL_EXPORTER_OTLP_ENDPOINT" \
```

Comment:
Since we are using logging here I think we can can export OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED=true and then update docker to forward it